app/controllers/graphql_controller.rb

```diff
@@ -21,30 +21,28 @@ def execute
 
   private
 
-  # Handle variables in form data, JSON body, or a blank value
-  def prepare_variables(variables_param)
-    case variables_param
+  # Handle form data, JSON body, or a blank value
+  def ensure_hash(ambiguous_param)
```

Comment:
You'll have to get me up to speed on this.

Reply:
I have no clue what this is! This change must've happened under the hood

Reply:
I thought you'd whipped up some "rails magic" for us! hahaha